src/js/components/App.vue

```diff
@@ -304,9 +501,108 @@ export default {
+            modalName: '',
+        },
+
+        birthdayMonthModal : [],
+
+        allBirthdayModal : [],
+
         sections: [
+            {
+                title: 'Birthdays',
+                type: 'birthday',
+                items: [
+                    {
+                        name: 'Barbara Carvalho',
+                        date: '22-03',
+                    },
+                    {
+                        name: 'Bruna Rubio',
+                        date: '18-03',
+                    },
+                    {
+                        name: 'Carlos Bastos',
+                        date: '27-12',
+                    },
+                    {
+                        name: 'Daniel Polito',
+                        date: '06-02',
+                    },
+                    {
+                        name: 'Douglas Gálico',
+                        date: '06-02',
+                    },
+                    {
+                        name: 'Eduardo Bojikian',
+                        date: '02-12',
+                    },
+                    {
+                        name: 'Eduardo Redressa',
+                        date: '06-04',
+                    },
+                    {
+                        name: 'Fabrício Souza',
+                        date: '10-01',
+                    },
+                    {
+                        name: 'Fellipe Rocha',
+                        date: '',
+                    },
+                    {
+                        name: 'Fernando Hideo',
+                        date: '03-04',
+                    },
+                    {
+                        name: 'Gabriel Oliveira',
+                        date: '17-11',
+                    },
+                    {
+                        name: 'Juliana Ferreira',
+                        date: '11-10',
+                    },
+                    {
+                        name: 'Karoline Kimiko',
+                        date: '21-04',
+                    },
+                    {
+                        name: 'Leandro Brito',
+                        date: '13-07',
+                    },
+                    {
+                        name: 'Matheus William',
+                        date: '16-04',
+                    },
+                    {
+                        name: 'Matheus Solha',
+                        date: '24-08',
+                    },
+                    {
+                        name: 'Ricardo Carneiro',
+                        date: '23-02',
+                    },
+                    {
+                        name: 'Tiago Silva',
+                        date: '22-05',
+                    },
+                    {
+                        name: 'Vinicius Lopes',
+                        date: '08-02',
+                    },
+                    {
+                        name: 'Wesley Francisco',
+                        date: '19-10',
+                    },
+                ],
```

Comment:
Move this array to an external `.json` file and import it here. It will keep things smaller here

Reply:
Also, this should be hidden, since this is a public project 😄